Backend/lib/scheduler/src/Lib/Scheduler/Handler.hs

```diff
@@ -65,7 +65,7 @@ handler hnd = do
     runTask anyJob@(AnyJob Job {..}) = mask $ \restore -> do
       let jobType' = show (fromSing $ jobType jobInfo)
       expirationTime <- asks (.expirationTime)
-      withDynamicLogLevel jobType' . Hedis.withCrossAppRedis . Hedis.whenWithLockRedis (mkRunningJobKey id.getId) (fromIntegral expirationTime) $
+      withDynamicLogLevel jobType' . Hedis.withCrossAppRedis . Hedis.whenWithLockRedis (mkRunningJobKey parentJobId.getId) (fromIntegral expirationTime) $
```

Comment:
why this changed?

Reply:
this is some bug, it was supposed to be `parentJobId` looks like somehow got reverted so fixed it here.